web/src/lib/utils/helpers.ts

```diff
@@ -144,6 +144,18 @@ export const getUserNameOrEmail = (userData: UserData | undefined): string => {
   }
 };
 
+export const getNavBarBusinessTitle = (userData: UserData | undefined): string => {
+  const Config = getMergedConfig();
+  if (userData?.profileData.industryId === undefined || userData.profileData.legalStructureId === undefined) {
+    return Config.navigationDefaults.navBarGuestText;
+  }
+  if (userData?.profileData.businessName === undefined || userData?.profileData.businessName === "") {
+    return `${Config.navigationDefaults.navBarUnnamedBusinessText} [${userData?.profileData?.industryId}][${userData?.profileData?.legalStructureId}]`;
```

Comment:
This is not correct. This will render these on the screen as the IDs that we use to represent these values; not the way design wants them to show.

`industryId` would be something like "home-improvement-contractor" and it looks like the designs want us to use the industry name ("Home Improvement Contractor")

For `legalStructureId`, this code would make it show up like "limited-liability-company". We can't use the name here, because the name would be "Limited Liability Company (LLC)" and it looks like the designs want us to show the abbreviation (ie, just "LLC") so we'll have to add something to legal structures to support this.

Reply:
So I looked around a bit and it seems like we only have an abbreviator for LegalStructureId in the backend and it would need to be modified even more. And it seems like potentially good for a follow up,  as this entire case wasn't initially covered in the story. Thoughts?

For industryId is there already a function for Title case?